spec/Cabin/SingleSeat.vspec

```diff
@@ -111,8 +107,8 @@ Lumbar.Height:
   datatype: uint8
   type: actuator
   min: 0
-  max: 255
-  description: Lumbar support position. 0 = Lowermost. 255 = Uppermost.
+  unit: mm
```

Comment:
For all other "Height" signals for seats we use absolute position in millimeters, so I assume we want that here as well.

(I see it as a different generic topic if we shall prefer absolute (mm) or relative (percent) values for seat positions - I do not know how positions typically are communicated inside vehicles, by relative or absolute values)